packages/mobile/src/navigation/tabs/tabs.navigation.tsx

```diff
@@ -16,17 +16,30 @@ import {
 } from '~/hooks/hooks';
 import {
   HomeIcon,
-  ListIcon,
   LogInIcon,
   StarIcon,
   Text,
   UserIcon,
+  ListIcon,
+  ButtonText,
+  BurgerMenu,
   Image,
 } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { StyleProp, ViewStyle } from 'react-native';
 import { WelcomeNavigation } from '../welcome/welcome.navigation';
 import { styles } from './styles';
 
 const Tabs = createBottomTabNavigator<MainNavigationParamList>();
+type TabOptions = {
```

Comment:
If you need partial type from BottomTabNavigationOptions, you can use `Pick<BottomTabNavigationOptions , 'option' | 'anotherOption']>`